python/ray/services.py

```diff
@@ -935,8 +935,9 @@ def _start_redis_instance(executable,
         load_module_args += ["--loadmodule", module]
 
     while counter < num_retries:
-        if counter > 0:
-            logger.warning("Redis failed to start, retrying now.")
+        if counter > num_retries // 2:
+            logger.warning("Redis failed to start after"
```

Comment:
Can we compare > 1 instead? It's a bit weird to warn for exactly half of the retries.

Reply:
Actually, `> 1` will still probably print the message. I remember this messages were printed 3~4 times.